config/chefstyle.yml

```diff
@@ -650,4 +650,9 @@ ChefRuby/GemspecRequireRubygems:
   Enabled: true
   VersionAdded: '1.3.0'
   Include:
-    - '**/*.gemspec'
\ No newline at end of file
+    - '**/*.gemspec'
+
+ChefRuby/RequireNetHttps:
+  Description: net/http is deprecated and just includes net/http and openssl. We should include those directly instead
```

Comment:
```suggestion
  Description: net/https is deprecated and just includes net/http and openssl. We should include those directly instead
```

?

Reply:
I won't bother commenting on the 5 other places this string appears in this PR

Reply:
fixed